Docs/docs/sdk/detecting-vrcsdk.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Detecting VRChat SDK"
+slug: "detecting-vrcsdk"
+hidden: false
+createdAt: "2023-10-31T07:18:15.860Z"
+updatedAt: "2023-10-31T07:18:15.860Z"
+---
+
+If your unity tools or libraries want to use APIs of VRChat SDK but not depends on VRChat SDK,
+there are several ways to detect VRChat SDK installation.
```

Comment:
```suggestion
There are several ways to detect the VRChat SDK in a Unity project. This can be helpful when developing Unity tools or libraries that do not depend on the VRChat SDK, but may still want to utilize [VRChat's SDK API](/sdk/public-sdk-api).
```